<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Places</class>
 <widget class="QMainWindow" name="Places">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Места</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QPushButton" name="btnConnect_4">
      <property name="text">
       <string>Присоединиться</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>343</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnSelectAll_4">
      <property name="text">
       <string>Выбрать все</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="lbAddress">
         <property name="text">
          <string>Адрес</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbCity">
         <property name="text">
          <string>Город</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="leCapacity"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="leAddress"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnDel_4">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="leCity"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnAdd_4">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbCapacity">
         <property name="text">
          <string>Вместимость</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QTableWidget" name="twPlaces"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTextEdit" name="teResult">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
